includes/REST_API/Stories_Controller.php

```diff
@@ -122,6 +120,24 @@ public function prepare_item_for_response( $post, $request ) {
 		return apply_filters( "rest_prepare_{$this->post_type}", $response, $post, $request );
 	}
 
+	/**
+	 * Updates a single post.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	public function update_item( $request ) {
+		$response = parent::update_item( $request );
+		if ( ! is_wp_error( $response ) ) {
+			// If publisher logo is set, let's assign that.
+			$publisher_logo_id = $request->get_param( 'publisher_logo' );
+			if ( $publisher_logo_id ) {
+				set_theme_mod( 'custom_logo', $publisher_logo_id );
```

Comment:
Updating the custom logo sounds very unexpected from a user perspective. The custom logo is used for many things, for example by the currently active theme. We don't want to deface the website by replacing the logo everywhere.

Should we perhaps rather use post meta for this? Or just store the ID as part of the story JSON blob?

Reply:
Ha, this was based on the logic in `get_publisher_logo` since that was the first value used there, however, now thinking about, I suppose `get_publisher_logo` was just with the purpose trying to get some kind of existing image and not assuming it would be used later :)

Yep, makes sense that we should use something else, especially since it seems to be story-based and not a global value.

Will change to use meta 👍

Reply:
I wonder if we should create a new image size for the publisher logo src, with a fallback to thumbnail size, and then fallback to original size. Thoughts?

Reply:
But a logo should be a global setting IMO. So it should be stored in an web stories option.

Reply:
@spacedmonkey Seems like [you're right](https://docs.google.com/document/d/1Xs4uOZ3KCNlTMdjDT6q7cvTUpA1xk-kzrnVLQALltQk/edit#).